### ISSUE TITLE
[Feature] Hotkey for disabling/enabling filters

### ISSUE THREAD

**Issue by necros2k7** [henrypp/simplewall#1716]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

3.7.8

### Problem you are trying to solve

Disabling/enabling filters with Win hotkey, not calling GUI

### Suggested solution

Add hotkey for app

**Comment by reddyshyam** [comment]
Yes, this would be a great addition to another request which is via command line.

**Comment by Ehab-d** [comment]
This feature will be very useful and will save time. I hope the developer implement this idea.
